pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -113,6 +119,7 @@ func calculateWeight(args framework.Arguments) priorityWeight {
 		podAffinityWeight:      1,
 		balancedResourceWeight: 1,
 		taintTolerationWeight:  1,
+		imageLocalityWeight:    0,
```

Comment:
Is this plugin disabled by default in k8s scheduler?

Reply:
Oh, I forgot to check how k8s deals with this. I thought we leave this decision to user, but I guess user will never overwrite the priority config. I will modify the weight.